portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -53,7 +77,33 @@ public Duration getDuration() {
     return duration;
   }
 
-  public Integer getPriority() {
+  public TaskPriority getPriority() {
+    return priority;
+  }
+
+  public Integer getPriorityInt() {
     return priority.getPriority();
   }
+
+  public long getDurationSeconds() {
```

Comment:
Consider removing this method, because the caller can get the duration and invoke getSeconds()

Reply:
Done.